elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/TableContext.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+
+package com.yahoo.elide.datastores.aggregation.metadata;
+
+import com.yahoo.elide.datastores.aggregation.query.Queryable;
+import com.github.jknack.handlebars.HandlebarsException;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * Context for resolving table arguments in provided expression.
+ */
+@Getter
+@ToString
+public class TableContext extends ColumnContext {
```

Comment:
What do we gain by extending ColumnContext here?  Can this just extend Map instead?